src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/InQueryGenerator.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search.Expressions.Visitors.QueryGenerators
+{
+    internal class InQueryGenerator : SearchParamTableExpressionQueryGenerator
```

Comment:
Is this still used/needed for now?

Reply:
Good question. 

I made these changes thinking based on the implementation of ChainLinkQueryGenerator and IncludeQueryGenerator. 

Should we then get rid of this class and return null in SearchParamTableExpressionQueryGeneratorFactory,VisitIn?

Reply:
Perhaps its ok to leave it for now if its needed as a default/passthrough for the visitors. I can't tell if its being called